registry/corestake/calldata-corestake.json

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "https://eips.ethereum.org/assets/eip-7730/erc7730-v1.schema.json",
+  "context": {
+    "contract": {
+      "abi": "https://openapi.coredao.org/api/contracts/abi_of_verified_contract/0xf5fA1728bABc3f8D2a617397faC2696c958C3409",
```

Comment:
This endpoint returns a HTTP 401 unauthorized. I can't access it and the CI probably can't either.